docs/snippets/tracer/captureAWSAll.ts

```diff
@@ -1,4 +1,4 @@
-import { Tracer } from '@aws-lambda-powertools/tracer';
+// import { Tracer } from '@aws-lambda-powertools/tracer';
 
-const tracer = new Tracer({ serviceName: 'serverlessAirline' });
-const AWS = tracer.captureAWS(require('aws-sdk'));
```

Comment:
We can convert this example to this:

```ts
import { Tracer } from '@aws-lambda-powertools/tracer';
import AWS from 'aws-sdk';

const tracer = new Tracer({ serviceName: 'serverlessAirline' });
tracer.captureAWS(AWS);
```

I have tested it and it works, plus it doesn't require any special treatment in terms of linting.

Reply:
with special treatment i meant efficient guidance from repo-maintainers , like you @dreamorosi  :) 
the option chosen is clear . The code-snippets are taking shape (form causality) to match the existing linting rules.
Aristoteles on [causality](https://en.wikipedia.org/wiki/Four_causes#Efficient)

i will do it.

Reply:
> The code-snippets are taking shape (form causality) to match the existing linting rules.

It's intentional 😉